assets/sass/components/global/_googlesitekit-tooltip.scss

```diff
@@ -19,3 +19,8 @@
 .googlesitekit-tooltip.MuiTooltip-tooltipPlacementBottom {
 	margin-top: 0;
 }
+
+// High z-index since the survey modal has a high z-index as well
+.googlesitekit-tooltip-popper.MuiTooltip-popper {
+	z-index: 99999;
```

Comment:
This is the same z-index used by the survey modal (and a few others that I found).
https://github.com/google/site-kit-wp/blob/37e9cad8e29c8fa83bd8337bf9403f80243d42a4/assets/sass/components/surveys/_googlesitekit-surveys-modal.scss#L23
The comment here seems to imply that this value should be higher than the survey modal but maybe that's just how I'm reading it. I'm guessing perhaps this was intentional but just want to confirm.

I think ideally, we should have a few variables for z-indexes and then others can be defined relative to those. It's also important to know what WP uses as well to help explain where things need to be relative to one of those. It's a little odd though that most of our z-indexes are less than `15`, but then we jump right to `10000` and `99999`. Changing all of these is not within the scope here but would be great to perhaps open an issue to address more holistically.

Reply:
@aaemnnosttv Actually the portal has the same z-index as the WordPress admin bar, which was intentional, just to make sure the portal content will always be on top of any content. Do you think we should lower the z-index? 99999 was like the safest value for me.

Reply:
Okay, that's fine then, just wanted to confirm. In that case, I think there's still room for improvement overall to use variables for the z-index rather than needing to add comments in all uses to explain, but that can be for another issue 👍